codes/oscillators/bosonic_rotation.yml

```diff
@@ -0,0 +1,29 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# code id, physical, logical are all lower case
+code_id: bosonic_rotation
+physical: oscillators
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Bosonic rotation'
+
+# First reference with the code (optional).
+introduced: '\cite{doi:10.1103/PhysRevX.10.011058}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'A bosonic rotation code encodes one qubit into a subspace of the oscillator such that the discrete rotation operator \(\exp\left(\mathrm{i} \pi \hat{n}/N \right)\) acts as the logical Pauli-Z operation. Here \(\hat{n}\) is the number operator diagonal in the Fock basis. The rotation symmetry ensures that encoded states have support only on every \(N^{\textrm{th}}\) Fock state, i.e. \(\sum_{j=0}^\infty \alpha_j \vert{j N}\) for some coefficients \(\alpha_j\).'
```

Comment:
\alpha is quatum optics convention for a coherent state; use "c". Add "\rangle" and also an offset, since support can be k modulo N for k nonzero. Could expand discussion a bit to explain that.

Reply:
qudit